### PR TITLE
Upgrade to latest nash (var keyword)

### PR DESCRIPTION
**Pull request by i4ki**

### CODE REVIEW

File: all.sh [hunk 1]
```diff
@@ -8,57 +8,85 @@ import history
 import files
 import programs
 
-NASHCOMPLETE_CMD = (
+var GOBIN = $GOPATH+"/bin"
+var ENZOTOOLS = ($GOBIN+"/ls" $GOBIN+"/echo" $GOBIN+"/cat")
```
Comment (vitorarins) [inline all.sh]: does this project depend on enzotools?

File: all.sh [hunk 2]
```diff
@@ -8,57 +8,85 @@ import history
...
+fn abort(msg) {
+	echo "error: "+$msg
+
+	exit(1)
+}
+
+fn init() {
+	for tool in $ENZOTOOLS {
+		if file_exists($tool) != "0" {
+			abort("Enzo tools not installed: "+$tool+" not found")
```
Comment (vitorarins) [inline all.sh]: I am not sure this should be required

File: all.sh (hunk 1 above)
Comment (i4ki) [reply all.sh]: yes

File: all.sh (hunk 2 above)
Comment (i4ki) [reply all.sh]: I'm not sure either =(
But this is only an attempt to make this project more reliable.

**Comment by i4ki** [comment]
I don't know why this merge conflict... I'll take a look into this soon.

**Comment by lborguetti** [comment]
PR #22 is same.

**Pull request closed without merging**